content/en/platform/corda/5.0-dev-preview-1/tutorials/building-cordapp/c5-basic-cordapp-flows.md

```diff
@@ -322,6 +336,7 @@ The `CreateBoardingTicket` flow lets Mars Express self-issue a `BoardingTicket`
 Now that you've written the `CreateAndIssueMarsVoucher` flow, try writing the `CreateBoardingTicket` flow.
 
 You will need these variables:
```

Comment:
@peterli-r3 Why and what they do? Are these variables to a function or to a class?

Reply:
These variables are part of the story. There is no why and what they do. They are just variables, that carry information. 

They are variables of the createboardingticket flow.

Reply:
yes, but what part of the business problem are they solving such that they need to be "on ledger" facts. Given days to launch is clealry temporal, why sign it to the ledger etc.

Reply:
Fair point. I will change it to the launch date. As flight ticket should have date&time.

Reply:
changed in code. updated tutorial.